plugins/woocommerce/client/blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/edit.tsx

```diff
@@ -15,88 +18,120 @@ import { ProductGalleryThumbnailsBlockSettings } from './block-settings';
 import { checkOverflow } from '../../utils';
 import type { ProductGalleryThumbnailsBlockAttributes } from './types';
 
-export const Edit = ( {
-	attributes,
-	setAttributes,
-}: BlockEditProps< ProductGalleryThumbnailsBlockAttributes > ) => {
-	const { thumbnailSize } = attributes;
-
-	const scrollableRef = useRef< HTMLDivElement >( null );
-	const [ overflowState, setOverflowState ] = useState( {
-		bottom: false,
-		right: false,
+const maxThumbnails = 10;
+const prepareProductImages = ( productImages: ProductResponseImageItem[] ) => {
```

Comment:
Lets add some comment docs for functions, pretty cheap and easy to do with AI so I think we should start doing this more.